timemachine/cpp/src/local_md_potentials.cu

```diff
@@ -55,7 +55,7 @@ LocalMDPotentials::LocalMDPotentials(
         default_bonds[i * 2 + 1] = i + 1;
     }
     std::vector<double> default_params(N_ * 3);
-    free_restraint_ = std::shared_ptr<FlatBottomBond<double>>(new FlatBottomBond<double>(default_bonds));
+    free_restraint_ = std::shared_ptr<FlatBottomBond<float>>(new FlatBottomBond<float>(default_bonds));
```

Comment:
Curiosity: what's the significance of this change?

Reply:
Previously, we were running our flat bottom restraint in double precision when it doesn't need to be. This was originally done because of a test that I thought was broken by this, but was actually because the system was not equilibrated with a barostat and had air bubbles in the system which caused issues.